packages/extension-ui/src/Popup/CreateAccount/index.tsx

```diff
@@ -18,9 +19,20 @@ export default function CreateAccount (): React.ReactElement {
   const [account, setAccount] = useState<null | { address: string; seed: string }>(null);
 
   useEffect((): void => {
-    createSeed()
-      .then(setAccount)
-      .catch((error: Error) => console.error(error));
+    getAccountCache()
+      .then((cachedAccount) => {
+        if (cachedAccount?.address) {
+          setAccount(cachedAccount);
+        } else {
+          createSeed()
+            .then((account) => {
+              setAccount(account);
+              setAccountCache(account).catch((e) => console.error(e));
+            })
+            .catch((error: Error) => console.error(error));
```

Comment:
Can remove this .catch, return the promise. (The outer catch will handle it)

Reply:
@typescript-eslint/no-floating-promises is complaining then (that's pretty new afaik):
> Promises must be handled appropriately or explicitly marked as ignored with the `void` operator.

Adding the void makes it complain for another reason
> Expected 'undefined' and instead saw 'void'.

Reply:
Weird. Then it would need to stay just to satisfy the linter. All ok.

(I'm not too fond of the "void" solution, the `.catch(console.error)` at least tells me I'm not setting anything, but at least am reporting on it.